main.go

```diff
@@ -78,6 +78,7 @@ func main() {
 	flag.BoolVar(&conf.FollowRedirects, "r", false, "Follow redirects")
 	flag.BoolVar(&conf.AutoCalibration, "ac", false, "Automatically calibrate filtering options")
 	flag.IntVar(&conf.Threads, "t", 40, "Number of concurrent threads.")
+	flag.IntVar(&conf.Timeout, "timeout", 10, "HTTP request timeout. Default: 10")
```

Comment:
The default is printed automatically in the CLI help so the `Default: 10` part can be removed.

Reply:
Oh, and we could mention that the unit is second, so something along the lines of
```
HTTP request timeout in seconds.
```

Though this is probably obvious for 99% of the users, it doesn't hurt to be more explicit.

Reply:
Done